src/node.rs

```diff
@@ -61,209 +80,283 @@ impl<Runtime> InternalNode<Runtime> {
 				.map(drop),
 		};
 
-		let config = cli
-			.create_configuration(&cli.run, TaskExecutor::from(task_executor))
-			.expect("failed to create node config");
-		let (task_manager, rpc_handlers) = build_node(config).unwrap();
+		let config = build_config(spec_factory, task_executor.into());
+
+		let (client, backend, keystore, mut task_manager) =
+			new_full_parts::<Node::OpaqueBlock, Node::RuntimeApi, Node::Executor>(&config)?;
+		let client = Arc::new(client);
+		let import_queue = manual_seal::import_queue(Box::new(client.clone()), &task_manager.spawn_handle(), None);
```

Comment:
Is there a way to avoid this initialization code? Can we plug into some existing bootstrap?

Reply:
well, we're setting up a node with manual seal, there's, unfortunately, no bootstrap available yet for setting up a node.